client/src/components/HomeHeader.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function HomeHeader({ creator }) {
+  return <h1 className="homeHeader"> Welcome, {creator} </h1>;
+}
+
+export default HomeHeader;
```

Comment:
React Components are great, but code should only be extracted to components if you expect to use it in more than one place. Since the welcome header is only going to be used in one place (the home page), this component isn't necessary. You can move the logic into `Home.jsx`